src/main/java/duke/ui/UI.java

```diff
@@ -0,0 +1,95 @@
+package duke.ui;
+
+import duke.datamanager.FileManager;
+import duke.task.Task;
+
+import java.io.IOException;
+
+public class UI {
+    public static String logo = " ____        _        \n"
+            + "|  _ \\ _   _| | _____ \n"
+            + "| | | | | | | |/ / _ \\\n"
+            + "| |_| | |_| |   <  __/\n"
+            + "|____/ \\__,_|_|\\_\\___|\n";
+
+    public static String divider = "-----------------------------------";
+
+    public static void showWelcomeScreen() {
+        System.out.println("Hello from\n" + logo);
+        System.out.println(divider);
+        System.out.println("Hello there! I'm Duke");
+        showLoadDataScreen();
+        System.out.println("I have the high ground! How may I help you?");
+        System.out.println(divider);
+    }
+
+    public static void showLoadDataScreen() {
+        System.out.println("Searching for saved data...");
+        try {
+            FileManager.readFile();
+            System.out.println("Loading saved data...");
+        } catch (IOException e) {
+            System.out.println("No data found - please ensure that a directory " +
+                    "called 'data' is in your project directory");
+        }
+    }
+
+    public static void showSaveDataScreen() {
+        System.out.println("Saving data...");
+        try {
+            FileManager.saveToFile();
+            System.out.println("Data saved successfully!");
+        } catch (IOException e) {
+            System.out.println("Unable to save - please create a directory called 'data' in the project directory!");
+        }
+    }
+
+    public static void requestInput() {
+        System.out.println(divider);
+        System.out.println("What is thy bidding, my master?");
+        System.out.println(divider);
+    }
+
+    public static void showAllTasks() {
+        System.out.println("This is your current list:");
+        for (int i = 0; i < Task.getTaskCounter(); i++){
```

Comment:
Try to avoid different levels of abstraction in a method. Refer to code quality on SLAP.

Reply:
Done! Thank you!